src/Core/Domain/Employee/ValueObject/Password.php

```diff
@@ -69,12 +82,24 @@ public function getValue()
     /**
      * @param string $password
      */
-    private function assertPasswordIsWithinAllowedLength($password)
+    private function assertPasswordIsWithinAllowedLength(string $password): void
     {
         $length = mb_strlen($password, 'UTF-8');
 
-        if (self::MIN_LENGTH > $length || $length > self::MAX_LENGTH) {
-            throw new EmployeeConstraintException(sprintf('Employee password length must be between %s and %s', self::MIN_LENGTH, self::MAX_LENGTH), EmployeeConstraintException::INVALID_PASSWORD);
+        if ($this->minLength > $length || $length > $this->maxLength) {
+            throw new EmployeeConstraintException(sprintf('Employee password length must be between %s and %s', $this->minLength, $this->maxLength), EmployeeConstraintException::INVALID_PASSWORD);
```

Comment:
```suggestion
            throw new EmployeeConstraintException(sprintf('Employee password length must be between %d and %d', $this->minLength, $this->maxLength), EmployeeConstraintException::INVALID_PASSWORD);
```

Are we sure it's employee here and not customer? Where does the string appear?

Reply:
Yes, it's sure. We are in the namespace of Employee, not Customer.

Reply:
Fixed